wordpress-mobile/WordPress-iOS
# Empty Stats: Fix presentation logic

Fixes [#17308](https://github.com/wordpress-mobile/WordPress-iOS/issues/17308)

This PR fixes the presentation logic.

## To test:
1. Delete the app and reinstall it (to clear the cache)
2. Go to My **Site** > **Stats** for site with less than 30 views
3. Complete all the Nudges (or dismiss them)
4. Return to **Site** > **Stats** screen and complete the Customize card (or dismiss it)
5. Nudges should be shown for other sites with less than 30 views
6. Customize card should not be shown for any site

## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
